src/system/ExtensionsModule/Api/VariableApi.php

```diff
@@ -27,7 +27,7 @@ class VariableApi
 {
     const CONFIG = 'ZConfig';
 
-    private $isInitialized = false;
+    private $isInitialised = false;
```

Comment:
please don't change this. I wrote the class, so I get to use my version of English.